corehq/ex-submodules/casexml/apps/stock/consumption.py

```diff
@@ -138,7 +138,8 @@ def get_transactions(case_id, product_id, section_id, window_start, window_end):
     SimpleTransaction = collections.namedtuple('SimpleTransaction', ['action', 'value', 'received_on'])
 
     def _to_consumption_tx(txn):
-        if txn.type in (const.TRANSACTION_TYPE_STOCKONHAND, const.TRANSACTION_TYPE_STOCKOUT):
+        if txn.type in (const.TRANSACTION_TYPE_STOCKONHAND, const.TRANSACTION_TYPE_STOCKOUT,
+                        const.TRANSACTION_TYPE_LA):
```

Comment:
In ILS they are completely ignored in consumption calculation(https://github.com/dimagi/rapidsms-logistics/blob/master/logistics/consumption.py#L67) so I leave the same behavior in commtrack. I have to add this here due to assertion below (https://github.com/dimagi/commcare-hq/pull/6403/files#diff-316b40eb9efd4d18615faac9aae10ed0L144)